### ISSUE TITLE
Contao 4.9.5 - Thumbnail won't display - Fix

### ISSUE THREAD

**Issue by lleitner-hub** [postyou/contao-abr-streaming-bundle#6]
Hello!

I had the issue:

- Want to create an abr-streaming element

- Choose my .m3u8 playlist

- Choose an thumbnail (.jpg or .png)
And my thumbnail won't get displayed on my website. 

I fixed it in the ce_abrstreaming.html5 template. There is a condition check against _poster_, but you need to work with _posterSRC_, like following:

```
<?php if ($this->posterSRC): ?>
	poster="<?php echo \FilesModel::findByUuid($this->posterSRC)->path; ?>" preload="none"
<?php endif; ?>
```

full template: 

```
<?php $this->extend('block_unsearchable'); ?>

<?php $this->block('content'); ?>
<!-- Überschrift -->
<?php if ($this->headline): ?>
<<?php echo $this->hl; ?>>
  <?php echo $this->headline; ?>
</<?php echo $this->hl; ?>>
<?php endif; ?>

<video class="abr-streaming<?php if ($this->autoplay): ?> autoplay<?php endif; ?>" 
	<?php if ($this->posterSRC): ?>
	poster="<?php echo \FilesModel::findByUuid($this->posterSRC)->path; ?>" preload="none"
	<?php endif; ?>
  <?= $this->size ?> controls playsinline webkit-playsinline>
  <?php foreach ($this->files as $file): ?>
  <source type="<?= $file->mime ?>" src="<?= $file->path ?>" title="<?= $file->title ?>">
  <?php endforeach; ?>
</video>

<?php $this->endblock(); ?>
```

Maybe someone can use this tip.

**Comment by MarkejN** [comment]
Hello! Thank you for pointing this out. I just released version 1.2.0 for Contao 4.9 and adjusted my content element to the current ContentMedia.php. This change adds new fields and should also fix this issue.

**Comment by lleitner-hub** [comment]
@MarkejN No problem :) Glad I could give you guys a hint.
